### PR TITLE
Additional tests for job_sort_key and node_sort_key for a multi sched

### PR DESCRIPTION
**Pull request by visheshh**
#### Bug/feature Description
Additional tests for job_sort_key and node_sort_key

#### Affected Platform(s)
ALL

#### Cause / Analysis / Design
few tests for multi sched were missing , Added as part of this PR

#### Solution Description
1. Test for job_sort_key set as ncpus for sc1
2. Test for node_sort_key set as ncpus for sc1

#### Testing logs/output
[ptl_output.txt](https://github.com/PBSPro/pbspro/files/1990519/ptl_output.txt)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_multi_sched.py [hunk 1]
```diff
@@ -1815,3 +1815,66 @@ def test_multisched_not_crash(self):
...
+        a = {'resources_available.ncpus': 1}
+        self.server.manager(MGR_CMD_SET, NODE, a, id='vnode[0]', expect=True)
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a, id='vnode[1]', expect=True)
+        a = {'resources_available.ncpus': 3}
+        self.server.manager(MGR_CMD_SET, NODE, a, id='vnode[2]', expect=True)
+        a = {'resources_available.ncpus': 4}
+        self.server.manager(MGR_CMD_SET, NODE, a, id='vnode[3]', expect=True)
+        a = {'resources_available.ncpus': 0}
+        self.server.manager(MGR_CMD_SET, NODE, a, id='vnode[4]', expect=True)
```
Comment (suresh-thelkar) [inline test/tests/functional/pbs_multi_sched.py]: Instead of making ncpus to 0 for a vnode which we are not interested please make it "offline".

**Comment by visheshh** [comment]
@suresh-thelkar , I have addressed your review comment , Please review.

**Comment by visheshh** [comment]
squashed and rebased , @PBSPro/pbspro-maintainers Please take this PR forward.

**Comment by bhroam** [comment]
I merged in the PR.

**Pull request merged**